core/Application/Kernel/EnvironmentValidator.php

```diff
@@ -75,6 +76,12 @@ private function checkConfigFileExists($path, $startInstaller = false)
             return;
         }
 
+        $general = Config::getInstance()->General;
```

Comment:
Does it work to access this here? This could be called before StaticContainer is initialized. Might be better to use `$this->settingsProvider`.

Reply:
@diosmosis done. It worked with the config as well but this way I suppose it is better 👍